packages/tailwind/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@vuestic/tailwind",
+  "description": "Script for config syncing between Tailwind CSS and Vuestic UI.",
+  "keywords": [
+    "vuesticui",
+    "vuestic",
+    "tailwindcss",
+    "tailwind",
+    "uikit",
+    "uiframework"
+  ],
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "./dist/sync.cjs",
```

Comment:
Small thing but if you use `tsx` to run script - it makes sense to skip build step and run `.ts` file directly.
If you compile file - you can use `npx`.

Reply:
Using `npx` can be problematic here:

<img width="834" alt="image" src="https://user-images.githubusercontent.com/64714442/207309995-c4216516-c79a-4069-8276-6a8c039c8f2e.png">

 I'd prefer to build it, because can't see any benefits of the other option.

Reply:
using node directly doesn't work either? why? o.o